src/Benchmarks.ServerJob/ServerJob.cs

```diff
@@ -33,7 +34,14 @@ public class ServerJob : IIdentifiable
 
         [JsonConverter(typeof(StringEnumConverter))]
         public Scheme Scheme { get; set; }
-
+        public string AspNetCoreVersion { get; set; }
+        public string RuntimeFrameworkVersion { get; set; }
+        public string Session { get; set; }
```

Comment:
Does the `ServerJob` need to know the `Session` and `Description`, or could these just live in the `Driver`?